client/package.json

```diff
@@ -37,5 +38,6 @@
     "not dead",
     "not ie <= 11",
     "not op_mini all"
-  ]
+  ],
+  "devDependencies": {}
```

Comment:
I think you meant to do `npm install --save-dev redux-devtools-extension`?

Reply:
Okay so I was a little confused about this, would this work in production then? I mean even if redux-devtools is not used it will still try and look for it in production because of the import

Reply:
Yeah I'm not totally sure about it either. I think we should make a staging environment with Heroku to test this out actually. I want to reduce the chances of us breaking major things in prod. 

https://github.com/zalmoxisus/redux-devtools-extension#13-use-redux-devtools-extension-package-from-npm

Reply:
Added an issue for this, I'll take care of it sometime later today or tomorrow.

https://github.com/GTBitsOfGood/bright-futures-academy/issues/125

Reply:
Oh alright! Actually would it possible to do it on a meeting day? I wanted to learn how to do this

Reply:
Actually it was really fast. If you log in to Heroku, you can see a bright-futures-academy-staging app. It's the same as prod for us. I'm currently deploying master branch to it to see how it turns out.

We should set up some best practices for it, but for now go ahead and just deploy your branch and check out how it works. If you want to tail the logs you can do that in the console or with `heroku logs --tail --app bright-futures-academy-staging`

Reply:
Oh okay! I'll try it out